SmartDeviceLink/SDLSendLocation.h

```diff
@@ -65,4 +64,25 @@
  */
 @property (strong, nonatomic) SDLImage *locationImage;
 
+/**
+  * Mode in which the sendLocation request is sent
+  *
+  * Optional
+  */
+@property (strong, nonatomic) SDLDeliveryMode *deliveryMode;
+
+/**
+   * Timestamp in ISO 8601 format
```

Comment:
Technically not in ISO 8601 format, though it should specify which optional portions are required for this.